src/main/java/rocks/inspectit/gepard/agent/internal/configuration/model/InspectitConfiguration.java

```diff
@@ -1,23 +1,21 @@
 package rocks.inspectit.gepard.agent.internal.configuration.model;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
 import rocks.inspectit.gepard.agent.internal.configuration.model.instrumentation.InstrumentationConfiguration;
 
 /** Model of an inspectit gepard configuration. */
 public class InspectitConfiguration {
 
+  private InstrumentationConfiguration instrumentation;
```

Comment:
In this case, we expect the json-property to be called "instrumentation", right? 
I think thats good:)
Nice to know: When I copied this model into the config-server, an empty InspectitConfiguration had two props on the root. instrumentation and instrumentationConfiguration. That was cause the get Method was called getInstrumentation, the prop was called instrumentationConfiguration and Jackson understood this wrong.